wallet/test/Test/Pos/Util/MnemonicOld.hs

```diff
@@ -106,7 +106,7 @@ numCS :: Int -> Entropy -> Integer
 numCS len = shiftCS . bsToInteger
   where
     shiftCS = case 8 - len `mod` 8 of
-        8 -> identity
+        0 -> identity
```

Comment:
hm, I see this change here as well https://github.com/haskoin/haskoin/blob/690c7ae8619f8f478e26cff08ada7bc24f758e8d/haskoin-core/src/Network/Haskoin/Crypto/Mnemonic.hs#L94 . Not sure how this affects things exactly

EDIT: I see https://github.com/haskoin/haskoin/blob/690c7ae8619f8f478e26cff08ada7bc24f758e8d/haskoin-core/src/Network/Haskoin/Crypto/Mnemonic.hs#L94 . All good

Reply:
See https://github.com/input-output-hk/cardano-sl/pull/3013